service/src/main/java/greencity/service/ubs/manager/BigOrderTableViewServiceImpl.java

```diff
@@ -50,6 +53,15 @@ public Page<BigOrderTableDTO> getOrders(OrderPage orderPage, OrderSearchCriteria
 
     @Override
     public void changeOrderTableView(String uuid, String titles) {
+        Employee employeeByUuid = employeeRepository.findByUuid(uuid).orElse(null);
+        if (nonNull(employeeByUuid)) {
+            TableColumnWidthForEmployee tableByEmployeeId = tableColumnWidthForEmployeeRepository
+                .findByEmployeeId(employeeByUuid.getId()).orElse(null);
+            if (nonNull(tableByEmployeeId) && tableByEmployeeId.isTableFreeze()) {
+                return;
```

Comment:
Maybe it's better to throw an exception instead of returning from the method, so that the global exception handler can catch it and send an appropriate HTTP status.